service/controller/app/v1/resource/status/resource.go

```diff
@@ -66,3 +67,22 @@ func New(config Config) (*Resource, error) {
 func (r Resource) Name() string {
 	return Name
 }
+
+// equals asseses the equality of AppStatuses with regards to distinguishing
+// fields.
+func equals(a, b v1alpha1.AppStatus) bool {
+	if a.AppVersion != b.AppVersion {
+		return false
+	}
+	if a.Release.LastDeployed != b.Release.LastDeployed {
```

Comment:
Should we compare `LastDeployed` as well? I can imagine that a situation that only this timestamp would have changed inside chart CR but anything else.

Reply:
The `LastDeployed` is the value we get from `helmclient` and that is also shown by `helm ls`. I don't see a downside to updating it if it changes.